ajax/libs/rsuite/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "rsuite",
+  "filename": "rsuite.min.js",
+  "version": "3.0.0-alpha",
```

Comment:
I see no `3.0.0-alpha` version on npmjs, maybe just update to the latest v3.0.1?